docs/source/new-docs/getting-started/installing/advanced-env-setup.rst

```diff
@@ -0,0 +1,155 @@
+:orphan:
+
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+Installing
+==========
+
+.. note::
+
+    Setting up WSL through windows is the easiest method for running GeoIPS on
+    a Windows system. To do so, follow directions on
+    `Microsoft WSL Install <https://learn.microsoft.com/en-us/windows/wsl/install>`_.
+    This requires administrator privileges.
+
+For all installation methods, you need to setup environmental variables.
+
+The required environmental variables are:
+
+  -``GEOIPS_PACKAGES_DIR`` is where GeoIPS looks for itself and its plugin packages
+  -``GEOIPS_TESTDATA_DIR`` is where GeoIPS looks to find its test datasets
+  -``GEOIPS_OUTDIRS`` defines where GeoIPS will write output data and imagery
+
+These are fine defaults for most users:
+
+.. code:: bash
+
+    export GEOIPS_PACKAGES_DIR=$HOME/geoips
+    export GEOIPS_TESTDATA_DIR=$GEOIPS_PACKAGES_DIR/test_data
+    export GEOIPS_OUTDIRS=$GEOIPS_PACKAGES_DIR/outdirs
+
+If desired, the GeoIPS environment variables can be added to your
+default shell configuration file (which is typically stored in
```

Comment:
Are we going to link to this from anywhere, or just leave it orphaned for now, for later update / completion?

Reply:
^ I think orphaned for now is fine